core-bundle/src/Image/Studio/FigureBuilder.php

```diff
@@ -281,7 +283,14 @@ public function fromUrl(string $url, array $baseUrls = []): self
             return $this;
         }
 
-        return $this->fromPath(urldecode(ltrim($path, '/')));
+        $path = urldecode(ltrim($path, '/'));
+
+        // Prepend the web_dir, except for files/ and assets/
+        if (!str_starts_with($path, $this->uploadPath.'/') && !str_starts_with($path, 'assets/')) {
```

Comment:
Do we need an exception for `/system/themes` or just `/system` too?

Reply:
I don't think so. Technically we also don't need it fore `assets/`. The only case where it is important is `files/` as otherwise the meta data would not get loaded. But that does not matter for any other folder.